app/vmagent/csvimport/request_handler.go

```diff
@@ -31,7 +31,7 @@ func InsertHandler(at *auth.Token, req *http.Request) error {
 	})
 }
 
-func insertRows(at *auth.Token, rows []csvimport.Row, extraLabels []prompbmarshal.Label) error {
+func insertRows(at *auth.Token, rows []csvimport.Row, extraLabels []prompb.Label) error {
```

Comment:
I think this function (and possibly others like it) can be simplified. Previously, we needed conversion logic due to different models, but now we can maybe just do `labels = append(labels, r.Tags...)`. 

That said, I'd prefer to handle this in a follow-up PR to keep this one focused on one thing and avoid growing its already non-trivial size.